miro_preprocessor/image_sorter/src/test_sorter_logic.py

```diff
@@ -5,6 +5,29 @@
 from sorter_logic import Decision, Undecidable, sort_image
 
 
+def collection_image_data(**kwargs):
+    image = {
+        'collection': 'source/images-A',
+        'image_data': {
+            "image_title": "Image Title",
+            "image_pub_title": "Image Pub Title",
+            "image_pub_periodical": "Lost socks monthly",
+            "image_library_dept": "Paperclips and hairnets",
+            "image_tech_captured_mode": "Frog retina",
+            "image_copyright_cleared": "N",
+            "image_access_restrictions": "CC-BY",
+            "image_general_use": "N",
+            "image_innopac_id": "12345678"
+        }}
+    collection = image['collection']
+    if 'collection' in kwargs.keys():
+        collection = kwargs.pop('collection')
```

Comment:
One slightly neater way of doing this might be to use the `.get()` method on dictionaries, which takes a key and a default:

```python
collection = kwargs.get('collection', 'source/images-A')
```

Then you could declare `image_data` directly, rather than through a level of dict structure.